### PR TITLE
templates: cover page and message flashing

### PR DESCRIPTION
**Pull request by lnielsen**
* Adds better styling of cover page.

* Fixes issues with message flashing not being included on e.g. cover
  pages.

Signed-off-by: Lars Holm Nielsen <lars.holm.nielsen@cern.ch>

### CODE REVIEW

File: invenio_theme/translations/da/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
-"Project-Id-Version: invenio-theme\n"
+"Project-Id-Version:  invenio-theme\n"
 "Report-Msgid-Bugs-To: info@invenio-software.org\n"
-"POT-Creation-Date: 2015-10-10 09:48+0200\n"
+"POT-Creation-Date: 2015-12-01 13:27+0100\n"
 "PO-Revision-Date: 2015-10-10 11:03+0000\n"
 "Last-Translator: larshankat <lars.holm.nielsen@cern.ch>\n"
-"Language-Team: Danish (http://www.transifex.com/zenodo/invenio-theme/language/da/)\n"
+"Language: da\n"
+"Language-Team: Danish (http://www.transifex.com/zenodo/invenio-"
```
Comment (jirikuncar) [inline invenio_theme/translations/da/LC_MESSAGES/messages.po]: zenodo vs inveniosoftware?

**Comment by lnielsen** [comment]
screen shot 2015-12-03 at 12 45 47 a m
screen shot 2015-12-03 at 12 47 13 a m

File: invenio_theme/static/scss/invenio_theme/cover.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
+/* Links */
```
Comment (jirikuncar) [inline invenio_theme/static/scss/invenio_theme/cover.scss]: license?

File: invenio_theme/templates/invenio_theme/body.html [hunk 3]
```diff
@@ -1,5 +1,3 @@
-
-{%- from "invenio_theme/macros/messages.html" import flashed_messages with context -%}
 <div class="container">
```
Comment (jirikuncar) [inline invenio_theme/templates/invenio_theme/body.html]: license?

**Comment by jirikuncar** [comment]
@lnielsen can you please check licence headers in templates and scss?

**Comment by lnielsen** [comment]
Ping @tiborsimko 
screen shot 2015-12-03 at 13 39 34
screen shot 2015-12-03 at 13 40 01

File: invenio_theme/templates/invenio_theme/macros/messages.html [hunk 4]
```diff
@@ -3,8 +3,10 @@
     {%- for category, msg in get_flashed_messages(with_categories=True) %}
```
Comment (jirikuncar) [inline invenio_theme/templates/invenio_theme/macros/messages.html]: licence?

**Comment by jirikuncar** [comment]
after https://github.com/inveniosoftware/invenio-theme/pull/24#discussion_r46570282 - :shipit:

**Comment by lnielsen** [comment]
Fixed

**Pull request merged**